tests/drivers/sensor/sbs_gauge/src/test_smartbattery.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021 Leica Geosystems AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __TEST_SMART_BATTERY_H_
+#define __TEST_SMART_BATTERY_H_
+
+const struct device *get_fuel_gauge(void);
+
+int32_t test_smartbattery_getSensorValue(int16_t channel);
```

Comment:
The return value doesn't seem to be used anywhere. Am I missing something? Why have it?

Reply:
there is no need for the return value, I did remove it